charts/tidb-operator/templates/controller-manager-rbac.yaml

```diff
@@ -1,68 +1,81 @@
 {{- if .Values.rbac.create }}
-{{- if .Values.clusterScoped }}
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: {{ .Values.controllerManager.serviceAccount }}
+  labels:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component: controller-manager
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+---
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
-  name: {{ .Release.Namespace }}:tidb-controller-manager
+  name: {{ .Release.Name }}:tidb-controller-manager
   labels:
-    app.kubernetes.io/name: {{ template "tidb-operator.name" . }}
+    app.kubernetes.io/name: {{ template "chart.name" . }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/component: controller-manager
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
 rules:
+{{- if .Values.clusterScoped }}
 - apiGroups: [""]
   resources:
     - services
     - events
   verbs: ["*"]
-- apiGroups: [""]
-  resources: ["nodes"]
-  verbs: ["get", "list", "watch"]
 - apiGroups: [""]
   resources: ["endpoints"]
   verbs: ["create", "get", "update"]
 - apiGroups: [""]
   resources: ["pods"]
   verbs: ["get", "list", "watch","update", "delete"]
-- apiGroups: [""]
-  resources: ["persistentvolumes"]
-  verbs: ["get", "list", "watch", "patch","update"]
-- apiGroups: [""]
-  resources: ["persistentvolumeclaims"]
-  verbs: ["get", "list", "watch", "update", "delete"]
 - apiGroups: ["apps"]
   resources: ["statefulsets"]
   verbs: ["*"]
 - apiGroups: ["pingcap.com"]
   resources: ["tidbclusters"]
   verbs: ["*"]
+{{- end }}
+- apiGroups: [""]
+  resources: ["nodes"]
+  verbs: ["get", "list", "watch"]
+- apiGroups: [""]
+  resources: ["persistentvolumes"]
+  verbs: ["get", "list", "watch", "patch","update"]
+- apiGroups: [""]
```

Comment:
The `persistentvolumeclaims` should be in the `{{ if .Values.clusterScoped }}` block.

I thought this was wrong at first glance. `nodes` and `persistentvolumes` are cluster wide resources and users cannot access these resources if not giving cluster scope permission. But I believe tidb-operator cannot work correctly without these permissions now. So users still needs cluster admin permission in order to create clusterrole/clusterrolebinding. This is a requirement whether user specify `clusterScoped` or not.